client/landing/stepper/declarative-flow/migration-signup.ts

```diff
@@ -56,9 +56,9 @@ const migrationSignup: Flow = {
 			}
 
 			if (
-				[ STEPS.SITE_CREATION_STEP.slug, STEPS.PROCESSING.slug, STEPS.ERROR.slug ].includes(
-					currentStep
-				)
+				currentStep === STEPS.SITE_CREATION_STEP.slug ||
+				currentStep === STEPS.PROCESSING.slug ||
+				currentStep === STEPS.ERROR.slug
```

Comment:
This is a logic change because TS doesn't like `.includes(`

Reply:
More info [here](https://stackoverflow.com/a/76415481)